cmd/kusk/cmd/upgrade.go

```diff
@@ -172,3 +187,22 @@ func init() {
 	upgradeCmd.Flags().StringVar(&releaseNamespace, "namespace", "kusk-system", "namespace to upgrade in")
 	upgradeCmd.Flags().BoolVar(&installOnUpgrade, "install", false, "install components if not installed")
 }
+
+func getVersions(component, container string, deployment appsv1.Deployment) (latest string, current string) {
```

Comment:
Shouldn't you return the `error`s that could potentially occur in this function, or at least report them?